packages/markdown/parseMarkdown.js

```diff
@@ -361,6 +361,33 @@ function createRender(context) {
             )}\n</aside>`;
           },
         },
+        {
+          name: 'tabs',
+          level: 'block',
+          start(src) {
+            const match = src.match(/===/);
```

Comment:
Not sure to understand. You plan to add the possibility to write the following markdown, but I do not see how it should be used

```md
===
Some content
===
```

Reply:
Sorry, this should be reverted.